glide.yaml

```diff
@@ -87,3 +87,11 @@ import:
   version: v3.3.1
   subpackages:
   - api
+- package: github.com/jenkins-x/go-scm
+  version: v1.5.102
+- package: github.com/shurcooL/githubv4
+- package: github.com/shurcooL/graphql
+- package: github.com/h2non/gock
```

Comment:
This shouldn't be necessary if you use the fake mechanism.

I see that you opened a PR in the fake driver to add additional functionality.

Reply:
@bigkevmcd We do have gock tests as well.   Gock tests cover the aspects of the testing that Fake client can't do.   It cuts deeper into the stack.   If someone broken github driver for example, Fake client won't help there.  You can't say having more test coverage is wrong.

Reply:
It's testing the wrong thing tho', it's testing go-scm, perhaps contributing the tests upstream?

Reply:
To be clear, it's a cost to maintain these, and we would need to double them if we add support for GitLab (and others).

Reply:
Of course, It is testing the right thing.   The tests invoke our methods.

Reply:
It is a small package.    The cost is less than typing these messages.